host_core/lib/host_core/wasmcloud/runtime_server.ex

```diff
@@ -145,6 +145,24 @@ defmodule HostCore.WasmCloud.Runtime.Server do
     {:noreply, state}
   end
 
+  @impl true
+  def handle_info({:perform_actor_log, claims, level, text, token}, state) do
+    text = "[#{claims.public_key}] #{text}"
+    actor = claims.public_key
+
+    case level do
```

Comment:
Couldn't this be expressed as an enum sent from the NIF?

Reply:
it could have, but I still have to run the case/match so sending atoms (enums w/no data) doesn't buy us anything